universal/data/nyse_o.csv

```diff
@@ -1,4 +1,4 @@
-A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,[,\,],^,_,`,a,b,c,d
+A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,x7,R,S,T,U,V,W,X,Y,Z,x1,x2,x3,x4,x5,x6,a,b,c,d
```

Comment:
Same here, is this just for debugging or an actual change?

Reply:
Same here.